__tests__/app.test.js

```diff
@@ -8,12 +8,24 @@ beforeAll(() => seed(data))
 afterAll(() => db.end())
 
 describe('GET /api - controller set up correctly', () => {
-    test('200: should respond connected, when making a successful request', () => {
+    test('200: return an object which describes all available endpoints on the API', () => {
         return request(app)
         .get('/api/')
         .expect(200)
         .then(({body}) => {
-            expect(body.msg).toBe('Connected!')
+            const {endpoints} = body
+            const possibleEndpoints = Object.keys(endpoints)
+            possibleEndpoints.forEach((endpoint) => {
+                const keys = Object.keys(endpoints[endpoint])
+                expect(typeof endpoints[endpoint].description).toBe('string')
+                if (keys.includes('queries')){
+                expect(Array.isArray(endpoints[endpoint].queries)).toBe(true)
+                }
```

Comment:
Could improve the accuracy/make this more dynamic by requiring in the endpoints.json to your test file directly and expecting that the response.body is equal to that object